index.bs

```diff
@@ -686,16 +686,71 @@ dictionary MLOperandDescriptor {
 
 ### The MLOperand interface ### {#api-mloperand}
 
-An {{MLOperand}} represents an intermediary graph being constructed as a result of compositing parts of an operation into a fully composed operation.
+An {{MLOperand}} represents an intermediary graph edge being constructed as a result of compositing parts of an operation into a fully composed operation.
```

Comment:
I think the existing description is already accurate as an operand can indeed be a whole sub-graph and not just a specific graph edge.

Reply:
OK. Should we use mention this explicitly for clarity? (there is only this one occurrence of "intermediary graph" the spec).

> An {{MLOperand}} represents an intermediary graph (sub-graph or edge) being constructed as a result of compositing parts of an operation into a fully composed operation.

Or should we just leave it as it was?

Reply:
Please leave it as it was.